lua/kommentary/config.lua

```diff
@@ -71,6 +71,25 @@ function M.has_filetype(filetype)
     return M.config[filetype] ~= nil
 end
 
+function M.config_from_commentstring(commentstring)
+    if commenstring == "/*%s*/" then return default end
+
+    local placeholder = '%s'
+    local where = commentstring:find(util.escape_pattern(placeholder))
+
+    if not where then
+        return default
+    else
+        where = where - 1
+    end
+    if where + #placeholder  == #commentstring then
+        return { commentstring:sub(1, -#placeholder-1), false }
+    end
+
+    return { false, { commentstring:sub(1, where),  commentstring:sub(where + #placeholder + 1, -1) } }
+end
+
+
 --[[--
```

Comment:
```suggestion
        return {commentstring:sub(1, -#placeholder-1), false}
    end
    return {false, {commentstring:sub(1, where),  commentstring:sub(where + #placeholder + 1, -1)}}
end

--[[--
```

Make formatting consistent with the rest of the code.